server/app/forms/admin/BulkStatusUpdateForm.java

```diff
@@ -0,0 +1,42 @@
+package forms.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** A form for selecting multiple applications to be considered for a status update. */
+public final class BulkStatusUpdateForm {
+  private List<String> applicationsIds;
```

Comment:
Question: I thought applicationIds are `long`s, not `String`s. Is this correct?

Reply:
This was a tricky piece to get working. I tried with list<long>, it didn't work. Now tried again it worked. java is messing with me :)